app/forms/steps/dwp/benefit_type_form.rb

```diff
@@ -34,7 +34,7 @@ def persist!
           attributes.merge(attributes_to_reset)
         )
 
-        crime_application.update(confirm_dwp_result: nil)
+        crime_application.partner&.update(attributes_to_reset)
```

Comment:
If a user was to go back to the client benefit type page and select a passporting benefit when they had already entered partner benefit details, it would need to be reset